docs/guides/session-management/low-latency-session-management.mdx

```diff
@@ -0,0 +1,113 @@
+
+A session invalidates in these cases:
+
+1. When the user logs out;
+1. When it expires;
+1. When one wants to force a user to logout by calling [revokeSession](https://github.com/ory/kratos-client-go/blob/master/docs/V0alpha2Api.md#RevokeSession).
+
+## How to cache sessions
+
+Caching allows reusing previously retrieved or computed data efficiently. A cache is a high-speed data storage
+layer that stores a subset of data, typically transient. Future requests for that data are served up faster than
+is possible by accessing the data’s primary storage location.
+
+One of the most used strategies is to use the least recently used (LRU) cache. There are many options to use in a system:
+
+1. Embedded in-memory caches such as LevelDB, BerkeleyDB, Flashdb, etc.
+1. Using Key-Value storage such as Redis, Memcached, Aerospike cache, etc.
+
+Depending on requirements, one can choose any of the key-value storage available but consider this:
+
+1. Using embedded in-memory solutions can give one low latency. However, an application stores state can prevent horizontally scaling it.
+1. Key-value storages add network latency, but an application does not store any state so that one can scale it without any issues.
+
+:::warning
+
+Be careful with the TTL for cached sessions, and do not forget to remove the session once it expires or a user logs out. Users can still have access because of cached data, leading to a potential security breach.
+
+:::
+
+Let us take a look at following examples. We have session middleware that validates sessions and caches data. The logic is simple
+
+1. Try to get cached session from Redis;
+1. If a session is valid, return it and do not request Ory Cloud;
+1. If data does not persist, validate a session against Ory Cloud and cache it.
+
+## Network architecture
+
+1. `8080` port is used by the REST API
+1. `4000` port is used by [`ory proxy`](https://ory.sh/docs/guides/cli/proxy-and-tunnel) that manages cookies for local domain and proxies traffic to Ory Cloud
+
+Running Ory proxy
+
+```bash
+ORY_SDK_URL=https://playground.projects.oryapis.com ory proxy http://localhost:8080
+```
+
+## Using embedded cache storage
+
+This example uses [dgraph-io/ristretto](https://github.com/dgraph-io/ristretto) memory-bound
+Go cache.
+
+```mdx-code-block title="main.go"
+import embedded from '!!raw-loader!../../../code-examples/low-latency-sessions/embedded/main.go'
+
+<CodeBlock language="go">{embedded}</CodeBlock>
+```
+
+## Using Redis cache
+
+The example below shows how to cache sessions using [Redis](https://redis.io)
+
+```mdx-code-block title="main.go"
+import index from '!!raw-loader!../../../code-examples/low-latency-sessions/main.go'
+
+<CodeBlock language="go">{index}</CodeBlock>
+```
+The strategy above works for most cases when one has a high-throughput system.
+
+:::warning
+Caching can worsen the situation in the low-throughput systems because of [RTT](https://en.wikipedia.org/wiki/Round-trip_delay)
```

Comment:
When using abbreviated names of concepts/technologies, try to present them in full length first. Here you use RTT - use `round-trip time (RTT)` instead or skip the abbreviation altogether.